app/agents/voice/breeze_buddy/workflows/order_confirmation/websocket_bot.py

```
@@ -58,6 +62,28 @@
 load_dotenv(override=True)
 
 
+def _trace_tool_call(tool_name: str, args: dict = None):
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)